README.md

```diff
@@ -116,6 +116,8 @@ carthage update Box Result
 
 If the framework you want to add to your project has dependencies explicitly listed in a [Cartfile][], Carthage will automatically retrieve them for you. You will then have to **drag them yourself into your project** from the [Carthage/Build] folder.
 
+If the embedded framework in your project has dependencies to other frameworks you must list them in the [Cartfile][], and **link them to application target** (even if application target does not have dependency to that frameworks and never uses them).
```

Comment:
You have an app. It depends on `A.framework.`. `A.framework` depends on `B.framework`. Only `A.framework` needs to be listed in the app's `Cartfile` (`B.framework` will be built automatically), but both frameworks will need to be linked to the application target.